config/resources/knative-serving-0.7.0.yaml

```diff
@@ -0,0 +1,1851 @@
+apiVersion: v1
```

Comment:
I'd prefer this file to reside beneath `cmd/manager/kodata/knative-serving/0.7.0.yaml`. The original choice to put it in `deploy/resources` (since renamed `config/resources`) was totally arbitrary, and I'd rather we not deal with symlinks just to preserve that arbitrary choice. :)